app/views/groups/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="flex flex-col justify-center h-full">
+  <div class="bg-main flex justify-between text-white items-center h-12">
+    <%= button_to "<", destroy_user_session_path, method: :delete, class: 'text-xl mx-2'  %>
+    <h1 class="font-proxima-bold">GROUPS</h1>
+    <%= link_to ">", new_group_path, class: 'text-xl mx-2'  %>
+  </div>
+  <div class="flex-1">
+    <div class="flex flex-col gap-1">
+      <% @groups.each do |group| %>
+        <%= link_to group_entities_path(group), class: "grid grid-cols-4 gap-2 bg-white p-2 mb-2 h-24 md:h-full" do %>
+            <img src="<%= group.icon %>" class="col-span-1 max-h-24">
+            <div class="col-span-3 flex items-center justify-between">
+                <p class="font-proxima-light text-text-25"><%= group.name %></p>
+                <p class="font-proxima-light text-text-50 font-bold">$<%= group.total %></p>
+            </div>
+        <% end %>
+      <% end %>
+    </div>
+  </div>
+  <div class="p-5 bg-secondary text-center">
+    <%= link_to 'Add group', new_group_path, class: 'font-proxima-light border rounded px-5 py-3 block w-full text-white' %>
+  </div>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=for%20that%20category.-,There%20is%20a%20button%20%22add%20a%20new%20category%22%20at%20the%20bottom,-that%20brings%20the), This button text suppose to be `Add a new category` instead of `Add group`

Reply:
Thanks @Omar-Muhamad  i am changing to match requeriments